src/eos-google-chrome-system-helper

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
+#
+# eos-google-chrome-system-helper: system helper to touch / reset the stamp file
+#
+# Copyright (C) 2017 Endless Mobile, Inc.
+# Authors:
+#  Mario Sanchez Prada <mario@endlessm.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import argparse
+import logging
+import os
+import sys
+
+
+STAMP_FILE_INITIAL_SETUP_DONE = '/var/lib/eos-google-chrome-helper/initial-setup-done'
+
+
+def exit_with_error(*args):
+    logging.error(*args)
+    sys.exit(1)
+
+
+def touch_file(file):
+    with open(file, 'a'):
+        os.utime(file, None)
+
+
+def create_stamp_file(filename):
+    dirname = os.path.dirname(filename)
+
+    try:
+        os.makedirs(dirname, exist_ok=True)
+        touch_file(filename)
+        logging.info("Stamp file created at {}".format(filename))
+    except OSError as e:
+        exit_with_error("Error creating stamp file at {}: {}"
+                        .format(filename, str(e)))
+
+
+def remove_stamp_file(filename):
+    if not os.path.exists(filename):
+        logging.info("No {} file found. Nothing to do".format(filename))
+        return
```

Comment:
Instead of checking this upfront, I would catch (and ignore) `FileNotFoundError` exceptions